src/sdam/monitor.ts

```diff
@@ -94,21 +92,17 @@ export class Monitor extends TypedEventEmitter<MonitorEvents> {
   connectOptions: ConnectionOptions;
   isRunningInFaasEnv: boolean;
   [kServer]: Server;
-  [kConnection]?: Connection;
+  connection: Connection | null;
```

Comment:
What is the motivation behind changing kConnection to connection? Or the backstory of why it was a symbol?

Reply:
In the pre-TS days of the driver, there were issues with users accessing driver internal properties because we had no way to mark them internal.  So we switched to using symbol properties, similar to how Nodejs handles its internal properties, because although they're still accessible to users, it's less accessible.

Now that we have TS with the `private` keyword, generated TS-doc that strips out anything with an `@internal` tag and someday JS private properties, we don't really need symbol properties anymore.  

We're slowly removing them because they cause headaches for us:

- harder to access in tests
- no cmd+click to jump to definition of a class property
- doc comments don't appear on class properties that use symbols when you hover over the property
- etc

Reply:
Got it! Thx for the context!